test/e2e/upstream-proxy.feature

```diff
@@ -7,7 +7,7 @@ Feature: UpstreamProxy
     Given a configuration with:
       """
       files = ['behind-proxy/plus.js', 'behind-proxy/test.js'];
-      browsers = ['ChromeHeadlessNoSandbox'];
+      browsers = ['ChromeHeadless'];
```

Comment:
Is this commit related to the previous one? Why this change?

Reply:
I tried to fix the issue with "Error: socket hang up" in e2e test
Builds like 
https://travis-ci.org/github/karma-runner/karma/jobs/650593759?utm_medium=notification&utm_source=github_status
https://travis-ci.org/github/karma-runner/karma/jobs/659054766?utm_medium=notification&utm_source=github_status

I am ok, if need to remove last commit. Do I need to do it? @johnjbarton

Reply:
Please create a PR for this change and remove this last commit. I will review and land the one related to e2e first.